huddleupui/src/layouts/CommunityLayout.jsx

```diff
@@ -148,295 +149,225 @@ export default function CommunityLayout({ children, allowedUserTypes, canNotMemb
-								{(communityInfo.memberType === 'owner' || communityInfo.memberType === 'moderator') && (
-									<Card title="Moderator Menu" style={{ boxShadow: "rgba(0, 0, 0, 0.1) 0px 4px 12px", marginBottom: 15 }}>
-										<Button style={{ backgroundColor: '#7952CC', fontWeight: 700, color: 'white' }} onClick={() => navigate(`/communities/${communityId}/settings`)}>
-											Community Settings
-										</Button>
-									</Card>
-								)}
-							</div>
-						) : (
-							<Card title="Users" style={{ boxShadow: "rgba(0, 0, 0, 0.1) 0px 4px 12px", marginBottom: 15 }}>
-								<Row justify="center">
-									<span>
-										You have to be a member to see users.
-										You don't have an invitation yet?
-										Check your <Link to='/invitations' style={{ color: "#7952CC" }}>invitations</Link>.
-									</span>
-								</Row>
-							</Card>
-						)}
-					</Sider>
-					): (
-					<>
-						<Button onClick={toggleDrawer} style={{ position: 'absolute', right: 10, top: 80, zIndex: 1000 }}>
-							<ContactsFilled style={{ fontSize: 21 }} />
-						</Button>
-						<Drawer
-							title="Community Sidebar"
-							placement="right"
-							onClose={toggleDrawer}
-							visible={drawerVisible}
-							width={300}
-						>
+					{screens.md ? (
+						<Sider width={300} style={{ background: 'transparent', borderTop: '1px solid #f0f0f0', marginRight: 20, marginTop: 20 }}>
 							{(communityInfo.memberType && (!communityInfo.isPrivate || communityInfo.memberType !== 'notMember') && communityInfo.memberType !== 'banned') ? (
 								<div>
-									{/* Repeating the same structure for mobile */}
-									<Card title="Description" style={{ marginBottom: 15 }}>
+									<Card title="Description" style={{ boxShadow: "rgba(0, 0, 0, 0.1) 0px 4px 12px", marginBottom: 15 }}>
 										<span>{communityInfo ? communityInfo.description : ''}</span>
 									</Card>
+									<Card title="Badges" style={{ boxShadow: "rgba(0, 0, 0, 0.1) 0px 4px 12px", marginBottom: 15 }}>
+										{communityInfo.badges && communityInfo.badges.length > 0 ? (
+											<>
+												<Row justify="center">
+													{communityInfo.badges.slice(0, 10).map(badge => (
+														<Col key={badge.id} span={8} style={{ textAlign: 'center', marginBottom: 10 }}>
+															<Tooltip title={badge.description}>
+																<Avatar 
+																	src={badge.image} 
+																	shape="circle" 
+																	size={64} 
+																	icon={!badge.image && <TrophyOutlined />} 
+																	style={{ filter: badge.userHasBadge ? 'none' : 'blur(4px)' }}
+																/>
+																<div>{badge.name}</div>
+															</Tooltip>
+														</Col>
+													))}
+												</Row>
+												{communityInfo.badges.length > 10 && (
+													<Row justify="center">
+														<span style={{ color: 'blue', cursor: 'pointer' }} onClick={() => setShowMoreBadges(true)}>Show More Badges</span>
```

Comment:
Did you define setShowMoreBadges function in somewhere? it gives error 'setShowMoreBadges' is not defined

Reply:
you are right, fixed